CraftTweaker2-API/src/main/java/crafttweaker/api/event/PlayerVisibilityEvent.java

```diff
@@ -0,0 +1,16 @@
+package crafttweaker.api.event;
+
+import crafttweaker.annotations.ZenRegister;
+import stanhebben.zenscript.annotations.ZenClass;
+import stanhebben.zenscript.annotations.ZenGetter;
+import stanhebben.zenscript.annotations.ZenMethod;
+
+@ZenClass("crafttweaker.event.PlayerVisibilityEvent")
+@ZenRegister
+public interface PlayerVisibilityEvent extends IPlayerEvent {
+  @ZenMethod
+  void modifyVisibility (double modifier);
```

Comment:
The default Event implementation is currently:

```java
        public void modifyVisibility(double mod)
        {
            visibilityModifier *= mod;
        }
```

As it's specifically a modifier *of the value of the modifier*, I felt it would be more confusing to have it as a getter/setter combo -- as you would set the value to something, then get the value only to get a different value.